handlers/auth.ts

```diff
@@ -25,6 +34,14 @@ const addProviderSchemas = [
 // Make sure they are sorted.
 addProviderSchemas.sort();
 
+const api = await getApi();
+api.registry.register({
+  ClaimHandle: {
+    baseHandle: "Bytes",
+    expiration: "u32",
+  },
+});
+
```

Comment:
1. I don't think this is doing what you want it to be doing.
2. You shouldn't need to register any new types.
3. Odd place to put it outside of any function.

Reply:
Thanks, I haven't worked with this before so probably missing something obvious.

For signatureVerify, it looks like I need to have access to the encoded but unsigned payload `claimHandlePayload` as used in:
```
    const { isValid } = signatureVerify(
      claimHandlePayload.toU8a(),
      hexToU8a(c.request.requestBody.handleSignature),
      publicKey
    );
```

Is there a way I can get this value from the `Submittable` object (output of `api.tx.handles.claimHandle(...)`)?

I can manually construct it without registering the codec by concatenating `handleBytes.toU8a()` with a 4-byte little-endian encoding of the expiration block number, but I feel like there's got to be an easier way.

Reply:
This works:
```
    const claimHandlePayload = new Uint8Array([
      ...handleBytes.toU8a(),
      ...numberToU8a(c.request.requestBody.expiration, 32).reverse(),
    ]);

    const { isValid } = signatureVerify(claimHandlePayload, hexToU8a(c.request.requestBody.handleSignature), publicKey);
```

That seems fragile though, which is why I had it generating with the same registry codec that the front end uses before the signature request (`login/signing.ts`).

An alternative solution (if there's not another way to regenerate claimHandlePayload from a raw object) would be to change CreateIdentityRequest (the request body structure for `/v1/auth/create`) to include the u8a encoded form of `{ baseHandle, expiration }`, even if that's a bit redundant.